avaframe/in3Utils/logUtils.py

```diff
@@ -0,0 +1,48 @@
+""" Defining a writable object to write the config to the log file"""
+import logging
+import logging.config
+import os
+from datetime import datetime
+
+def dumpCfg(cfg, cfgName='Unspecified'):
```

Comment:
I am personally not found of the word 'Dump'. I find it a bit aggressive and code should be neutral. I think we should use something like Cfg2Log or writeCfg for example.